.goreleaser.yml

```diff
@@ -0,0 +1,27 @@
+builds:
+- binary: control-api
+  env:
+    - CGO_ENABLED=0 # this is needed otherwise the Docker image build is faulty
+  goarch:
+    - amd64
```

Comment:
Maybe we could add `arm64` too? 😊

Reply:
Why? Any controller or API server we build will run exclusively on OpenShift on x86

Reply:
Is there never a need for a local setup?

Reply:
It is, but not with Goreleaser. Goreleaser is meant purely for releasing in GH actions.
I guess I'll need to re-add GOOS and GOARCH to Makefile.

Reply:
I've added instructions in README how to build with ARM/mac